src/encoded/static/scss/encoded/modules/_static-pages.scss

```diff
@@ -533,7 +533,259 @@
 	}
 }
 
+/**********************************************************
+		Markdown Sortable Table
+***********************************************************/
+$markdown-table-edge-shadow-color : rgba(0,0,0,0.125) !default;
+$markdown-table-result-row-height: 46px !default;
+
+@mixin markdown-table-column-and-header-block {
+	//float: left;
+	//transform: translate3d(0,0,0);
+	> .inner {
+		height : 100%; //$markdown-table-result-row-height;
+		min-height: $markdown-table-result-row-height;
+		padding: 0 8px;
+		line-height: 16px;
+		text-align: center;
+		overflow: hidden;
+		text-overflow: ellipsis;
+        white-space: nowrap;
+        display: flex;
+		align-items: center;
+		line-height: $line-height-base;
+	}
+	> .width-adjuster {
+		position: absolute;
+		color: $search-results-header-text-color;
+		padding-left: 10px;
+		padding-right: 15px;
+		left: 0;
+		top : 0;
+		margin-left: -12px;
+		z-index: 10;
+		cursor: col-resize;
+		width: 28px;
+		height: 100%;
+
+		&:after {
+			content: ' ';
+			border-left-width: 1px;
+			border-right-width: 1px;
+			border-left-style: dotted;
+			border-right-style: dotted;
+			border-color: $search-results-header-text-color;
+			opacity: 0.6;
+			width: 3px;
+			left: 10px;
+
+			top: 9.5px;
+			bottom: 9.5px;
+			position: absolute;
+		}
+		&:hover:after {
+			opacity: 1;
+		}
+	}
+}
+
+@mixin markdown-table-column-block {
+    @include markdown-table-column-and-header-block;
+
+    > .inner > .value {
+        // Default style -- can be overriden
+        @include text-ellipsis-container;
+		flex-grow: 1;
+		font-size: 0.9rem;
+
+		img {
+			padding: 3px;
+		}
+    }
+}
+
+.markdown-table-outer-container {
+
+	.markdown-table-container {
+		position: relative;
+
+		.shadow-border-layer {
```

Comment:
I don't think this is used in the Markdown table and could potentially be removed?
Much of this style is likely to be deprecated very soon in any case since had to modify SearchResultTable CSS for other load-as-you-scroll tables as part of some refactoring in setting up EmbeddedSearchViews. Probably worth looking at after both this PR and the EmbeddedSearchView are merged.

Reply:
@alexkb0009 .shadow-border-layer rule/block is redundant. Moved it, thanks.